docs-src/getting-started.md

```diff
@@ -0,0 +1,55 @@
+# Getting started with Services APIs
+
+## Installing CRDs
+
+Service APIs is a collection of Kubernetes Custom Resource Definitions that need
+to be installed into any Kubernetes cluster.
+
+To install the CRDs, please execute:
+
+```
+kubectl kustomize "github.com/kubernetes-sigs/service-apis/config/crd?ref=master" \
+| kubectl apply -f -
```

Comment:
Is there a good way to do this without kustomize? It seems like we're not really using kustomize for anything. (This does not need to block this PR).

Reply:
We do this in our Makefile too: https://github.com/kubernetes-sigs/service-apis/blob/master/Makefile#L54

Do note that the user doesn't need to install `kustomize`. It is included as a plugin inside `kubectl` itself. Users need to make sure that their `kubectl` version is not an ancient one, which again is expected.